### PR TITLE
Prepare release

### LINKED ISSUE #263: Fuzzer doesn't work with modules
Before prepping the release, I tried to run the json fuzzer and ran into this:
https://github.com/dvyukov/go-fuzz/issues/195

### PR DESCRIPTION
**Pull request by liamsi**
- update changelog
- fix fuzzer via makefile (closes #263)
- bump versions

cc @alexanderbez @fedekunze 
(As a side note: we should probably get rid of the master/develop dichotomy here too).

Ran the JSON fuzzer for over 30 mins. No crashers 🤞

### CODE REVIEW

**Review by alexanderbez — APPROVED** [review]

**Comment by liamsi** [comment]
Need approval by someone with write access here (cc @ebuchman @jaekwon @mossid )

**Pull request merged**
